charts/refarch-templates/templates/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+{{- $dot := . }}
+{{- range $configmap := .Values.configMaps -}}
+{{ $data := dict "dot" $dot "module" $configmap }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "getFullname" $data }}
+  labels:
+    {{- include "getLabels" $data | trim  | nindent 4 }}
+data:
+{{- range $key, $value := $configmap.data }}
```

Comment:
hier kann man sich wieder Streiten, wir haben extra alles in das Deployment überführt mit den Umgebungsvariablen

Reply:
@ejcsid soll es entscheiden.

wenn wir das haben bitte checksum in deployment einbauen https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments

Reply:
> Ich würde beides anbieten.

Da ich ja entscheiden soll: können schon beides drin lassen, aber in Beispielen besser die Umgebungsvariable nehmen wie wir es jetzt haben.

Die Checksum sollte noch rein.